modulus/experimental/resdiff/training/loss.py

```diff
@@ -204,14 +203,14 @@ def __init__(self, P_mean=0.0, P_std=1.2, sigma_data=0.5):
         self.P_std = P_std
         self.sigma_data = sigma_data
         
-        print('dist.get_rank', dist.get_rank)
-        if dist.get_rank == 0:
-            breakpoint()
+        # print('dist.get_rank', dist.get_rank)
+        # if dist.get_rank == 0:
+        #     breakpoint()
          
         with torch.no_grad():
             #resume_state_dump = '/lustre/fsw/nvresearch/mmardani/output/logs/edm_era5_cwb_12chans_fcn_4x_crop448_zarr_fulldata_unetregression/complex-mayfly_2023.04.25_16.42/output/training-state-006924.pt'
             resume_state_dump = '/lustre/fsw/nvresearch/mmardani/output/logs/edm_era5_cwb_12chans_fcn_4x_crop448_zarr_fulldata_unetregression/mindful-parakeet_2023.04.03_22.54/output/training-state-042650.pt'
-            dist.print0(f'Loading training state from "{resume_state_dump}"...')
+            print(f'Loading training state from "{resume_state_dump}"...')
```

Comment:
Will this print for every rank?

Reply:
Yes, but once the logger is implemented I'll fix this to print on rank zero only